### DISCUSSION TITLE
Using tags input in laravel livewire

### DISCUSSION THREAD

**Discussion by penafarhan** [livewire/livewire#2521] [1. Help]
I just used and modified code from the example of tags input that provided in example folder [this](https://github.com/alpinejs/alpine/blob/master/examples/tags.html) Can anyone help me how to get the data from the input tags into the livewire. Thanks in advanced.

**Comment by rezahmady** [comment]
in your blade :

```
x-data="
{
tags: @entangle('tags') ,
newTag: '' 
}
"
```

in livewire controller define this property:
```
public $tags = [];
```

**Comment by MCKLtech** [comment]
Is this example still available? I get a 404 with the link.
